docs/README.md

```diff
@@ -19,10 +19,10 @@ sections:
     desc: Discover how to build modules for the Cosmos SDK.
     icon: modules
     url: /building-modules/intro.html
-  - title: Interfaces
-    desc: Build interfaces for Cosmos SDK applications.
+  - title: Running a Node
+    desc: Running and interacting with nodes using the CLI and API.
     icon: interfaces
-    url: /interfaces/interfaces-intro.html
+    url: /run-node
```

Comment:
Does this change make sense? This is the broken link referenced in #9441.

Reply:
it makes sense to me